src/MongoDB/Driver/WriteConcern.cpp

```diff
@@ -23,9 +23,6 @@
 #include "../../../utils.h"
 
 #include "../../../libmongoc/src/mongoc/mongoc-write-concern.h"
-#define MONGOC_I_AM_A_DRIVER
-#include "../../../libmongoc/src/mongoc/mongoc-write-concern-private.h"
-#undef MONGOC_I_AM_A_DRIVER
```

Comment:
This is a good idea. I should be removing the private includes as I go through files.